### DISCUSSION TITLE
Does GitHub Copilot interfere with C# Dev Kit?

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#109928] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Today I started my free trial with GitHub Copilot. Previously I installed C# Dev Kit in Visual Studio Code. Once I restarted VS Code I got a message saying that the C# Dev Kit would not run because GitHub Copilot was activated. So, should I remove C# Dev Kit? I tried saerching for the answer to this question, but didn't get any useful responses.

**Comment by timheuer** [comment, marked as answer]
tl;dr - no you don't need to uninstall C# Dev Kit ;-)

HI Rod! The message you likely got was from IntelliCode. A new version of IntelliCode extension was released last week and you should be able to get the update.  The message was a bit confusing really.  But what it was trying to convey was that if you had Copilot enabled and also IntelliCode, that IntelliCode wouldn't try to do the 'whole line completions' and yield that work to Copilot.

If you don't want starred completions in intellisense (the other capability IntelliCode provides), you can uninstall that extension completely (I personally love that feature).  Starred completions don't interfere with Copilot though.

make sense?
